### DISCUSSION TITLE
bvnc

### DISCUSSION THREAD

**Discussion by praveenkumar222** [Yisus7u7/termux-desktop-xfce#56] [Q&A]
![Screenshot_20211114-165220](https://user-images.githubusercontent.com/77523501/141680500-8f7023f8-f8ed-4745-a3db-e144e49b117c.jpg)
How can I use tigervnc ?

**Comment by JesusChapman** [comment]
You must switch it to ultravnc mode

**Reply by praveenkumar222** [reply]
How ?

**Comment by ghost** [comment]
how ?

Pada tanggal Min, 14 Nov 2021 22.00, Yisus7u7 ***@***.***>
menulis:
